src/ngrest/base/NgRestModel.php

```diff
@@ -109,14 +109,31 @@ public function extraFields()
         return array_merge(parent::extraFields(), $this->extractRootFields($extraFieldKeys));
     }
     
+    /**
+     * Attach behaviours to ever new {\luya\admin\ngrest\base\NgRestActiveQuery} on find() and ngRestFind().
+     *
```

Comment:
Please add more usage infos here, personally i never came into the situation where i needed to attach a behavior to the ActiveQuery class, so it would be nice to make examples and explain more when this could be useful. As far as i remember there is only a single event which could be taken for attaching stuff, but of course adding further methods (which is, i think your use case?) might be helpful.

Reply:
For linking purposes use doubled curly brackets: `{{ .... }}` otherwise this won't work on luya.io